audit-worker-versions/main.go

```diff
@@ -133,21 +168,67 @@ func createTasks(queue *tcqueue.Queue, taskGroupID string) {
 }
 
 func inspect(queue *tcqueue.Queue, taskIDs []string) {
-	for _, taskID := range taskIDs {
+	workers := make([]WorkerInfo, len(taskIDs))
+
+	for i, taskID := range taskIDs {
 		statusResponse, err := queue.Status(taskID)
 		if err != nil {
 			fmt.Println("Could not get status for task " + taskID)
 			panic(err)
 		}
-		workerPoolID, versionInfo := show(queue, statusResponse)
+		workerPoolID, workerInfo := show(queue, statusResponse)
+		versionInfo := workerInfo.info()
+		workers[i] = workerInfo
 		filename := FilenameEscape(workerPoolID)
 		err = ioutil.WriteFile(filename, append([]byte(versionInfo), '\n'), 0644)
 		if err != nil {
 			log.Fatalf("Error:\n%v", err)
 		}
-		name := workerPoolID + ":                                                                          "
-		fmt.Print(name[:75])
-		fmt.Println(versionInfo)
+		fmt.Printf("%-70s %s\n", workerPoolID+":", versionInfo)
```

Comment:
```suggestion
		fmt.Printf("%-70s: %s\n", workerPoolID, versionInfo)
```

Reply:
this way `:` will be hanging somewhere on the right always and not stick to the worker pool id

Reply:
ahhhhh, that makes sense - sorry!